### PR TITLE
Add unsubscibe

### PR DESCRIPTION
**Pull request by yaacov**
**Description**

We currently do not have an unsubscribe  function.

### CODE REVIEW

File: pkg/connections/stomp/subscribe.go [hunk 1]
```diff
@@ -23,12 +25,21 @@ import (
 func (c *Connection) Subscribe(destination string, callback func(m client.Message, destination string) error) (err error) {
 	var subscription *stomp.Subscription
 
+	// Ceack if we already subscibe to this destination, we do not allow for
```
Comment (oourfali) [inline pkg/connections/stomp/subscribe.go]: typos

File: pkg/connections/stomp/subscribe.go [hunk 2]
```diff
@@ -40,3 +51,16 @@ func (c *Connection) Subscribe(destination string, callback func(m client.Messag
 
 	return
 }
+
+// Unsubscribe unsubscribes from a destination
+func (c *Connection) Unsubscribe(destination string) (err error) {
+	// Ceack if we already subscibe to this destination,
```
Comment (oourfali) [inline pkg/connections/stomp/subscribe.go]: typos

File: pkg/connections/stomp/subscribe.go [hunk 3]
```diff
@@ -40,3 +51,16 @@ func (c *Connection) Subscribe(destination string, callback func(m client.Messag
...
+	// We do not allow for multiple subscriptions for one destination.
```
Comment (oourfali) [inline pkg/connections/stomp/subscribe.go]: not sure that comment is relevant here.

File: pkg/connections/stomp/subscribe.go (hunk 2 above)
Comment (yaacov) [reply pkg/connections/stomp/subscribe.go]: :+1:  done

File: pkg/connections/stomp/subscribe.go (hunk 3 above)
Comment (yaacov) [reply pkg/connections/stomp/subscribe.go]: :+1: Thanks changed to:

```
check if we subscribe to this destination,o/w return an error.
```

**Pull request merged**